lib/logstash/codecs/plain.rb

```diff
@@ -24,24 +24,19 @@ class LogStash::Codecs::Plain < LogStash::Codecs::Base
   # This only affects "plain" format logs since json is `UTF-8` already.
   config :charset, :validate => ::Encoding.name_list, :default => "UTF-8"
 
-  public
+  MESSAGE_FIELD = "message".freeze
+
   def register
     @converter = LogStash::Util::Charset.new(@charset)
     @converter.logger = @logger
   end
 
-  public
   def decode(data)
-    yield LogStash::Event.new("message" => @converter.convert(data))
-  end # def decode
+    yield LogStash::Event.new(MESSAGE_FIELD => @converter.convert(data))
```

Comment:
This change is not mentioned in the commit message. I'm in favor of this style (constant frozen string); can you update the commit message to include your reasoning?

Reply:
yep will do!

Reply:
done!